src/.env.sample

```diff
@@ -171,4 +171,7 @@ DEFAULT_REVIEWER_ROLE="reviewer"
 # Interface service host address 
 INTERFACE_SERVICE_HOST=http://localhost:3567
 
+# Regular expression to validate observation deep link
+OBSERVATION_DEEP_LINK_REGEX= "^https:\/\/dev\.elevate-ml\.shikshalokam\.org\/view\/observation\/[a-f0-9]{32}$"
```

Comment:
@adithyadinesh0412 Is this format is mentioned in story or get from the elevate survey

Reply:
I took it from elevate survey. @priyanka-TL . Anyway this will be tested in elevate initially and for each adoption we will have to modify this accordingly.